publish.gradle

```diff
@@ -30,24 +31,11 @@ publishing {
             }
         }
     }
-
-    repositories {
```

Comment:
We default to something now that we can remove the repo definition?

Reply:
Yes, from convention (oss) plugin